### PR TITLE
reworked: the movies send function in controller

### PR DESCRIPTION
**Pull request by Rasmus-Kibshede**
Switched from messagepattern (rabbitmq listener) to http request

### CODE REVIEW

File: src/Interface/Controllers/movie.controller.ts [hunk 1]
```diff
@@ -57,10 +57,17 @@ export class MovieController {
...
+    status: 200,
+    description: 'Gets all movies from database and sends them to ES via RabbitMQ',
+    type: '',
+    isArray: true,
+  })
   async getMoviesEs() {
-    const response = this.movieService.getMovies();
-    this.client.emit('movies_list', response);
+    const response = await this.movieService.getMovies();
+    this.client.emit('InsertMovies', response);
```
Comment (Geofery) [inline src/Interface/Controllers/movie.controller.ts:70]: - InsertMovies --> movies_list or insert_movies. naming convention rabbitMQ: https://www.rabbitmq.com/tutorials/tutorial-five-go

File: src/Interface/Controllers/movie.controller.ts [hunk 2]
```diff
@@ -57,10 +57,17 @@ export class MovieController {
     return this.movieService.getMovies();
   }
 
-  @MessagePattern('get_all_movies')
+  @Get('es')
+  @ApiOperation({ summary: 'Send all movies to ES' })
+  @ApiResponse({
+    status: 200,
+    description: 'Gets all movies from database and sends them to ES via RabbitMQ',
```
Comment (Geofery) [inline src/Interface/Controllers/movie.controller.ts:64]: Send all movies from database to ES, via RabbitMQ

**Review by Geofery — COMMENTED** [review]
See comments

File: src/Interface/Controllers/movie.controller.ts (hunk 1 above)
Comment (Rasmus-Kibshede) [reply src/Interface/Controllers/movie.controller.ts:70]: Nestjs naming convention, but we will use insert_movies

**Pull request merged**